apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -542,11 +542,11 @@
 <e>       <p><l>ich<s n="v"/><s n="tv"/></l>           <r>ish<s n="v"/><s n="tv"/></r></p></e>
 <e>       <p><l>ideologiya<s n="n"/></l>               <r>ideologiya<s n="n"/></r></p></e>
 <e>       <p><l>imperiya<s n="n"/></l>                 <r>imperiya<s n="n"/></r></p></e>
-<e>       <p><l>in<s n="n"/></l>                       <r>in<s n="n"/></r></p></e>
+<!-- <e>       <p><l>in<s n="n"/></l>                       <r>in<s n="n"/></r></p></e> -->
 <e>       <p><l>institut<s n="n"/></l>                 <r>institut<s n="n"/></r></p></e>
 <e>       <p><l>internet<s n="n"/></l>                 <r>internet<s n="n"/></r></p></e>
 <e>       <p><l>ion<s n="n"/></l>                      <r>ion<s n="n"/></r></p></e>
-<e>       <p><l>ish<s n="n"/></l>                      <r>ish<s n="n"/></r></p></e>
+<!-- <e>       <p><l>ish<s n="n"/></l>                      <r>ish<s n="n"/></r></p></e> existing word -->
```

Comment:
If the word already exists in the dictionary, you can just delete it—no need to comment it out.

Reply:
ok